static/pages/partnerships/Partnerships.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+oppiaFoundationWebsite.controller(
+  'PartnershipsPage', ['$scope', function($scope) {
+    var requestParams = {
```

Comment:
Better to just define this within submitContactUsForm and then do $http.post() with it to the relevant handler in the backend.

Reply:
I would still need like a Url handler right? something like "/mailhandler" that is served by main.py?

Reply:
Yes that's right. But you would serve it from main.py and you can define a path to that in app.yaml.

Given the structure of this site maybe define a custom url pattern for that in app.yaml along the lines of /ajax/* or something (put it above the general .* catchall). Then you can have /ajax/mailhandler etc.